common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -134,7 +135,8 @@ public class RemoteBlockPushResolver implements MergedShuffleFileManager {
   final DB db;
 
   @SuppressWarnings("UnstableApiUsage")
-  public RemoteBlockPushResolver(TransportConf conf, File recoveryFile) throws IOException {
+  public RemoteBlockPushResolver(
```

Comment:
@mridulm @zhouyejoe Sorry, I haven't found the entry of non test code of this constructor . Could you tell me where it is?

Reply:
Eureka ~ :)

Reply:
It is getting called through reflection in [YarnShuffleService.java](https://github.com/apache/spark/blob/master/common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleService.java#L325)

Reply:
thanks @zhouyejoe